src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -5,18 +5,18 @@
   "added": [
     {
       "what": [
-        "splice function in workflows that can change the contents of a collection of elements by removing or replacing existing elements and/or adding new elements in place."
+        "splice function in workflows that can change the contents of an array by removing or replacing existing elements of different types like strings, numbers, etc. and/or adding new elements in place."
```

Comment:
I hate `etc.`.

If you're listing things either:

* list everything (if the list is finite)
* categorise things (if the list is infinite) -- such as saying `we calculate even and odd numbers` instead of `we calculate 1, 2, 3, 4, 5, etc. numbers`

What's behind etc? Where do I need to read to know what etc hides?